src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import duke.command.Command;
+import duke.storage.Storage;
+import duke.task.TaskList;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+
+    public static void main(String[] args) {
+
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        String logo2 = "                   _      \n"
+                + "                  | |     \n"
+                + "  _   _ _ __   ___| | ___ \n"
+                + " | | | | '_ \\ / __| |/ _ |\n"
+                + " | |_| | | | | (__| |  __/ \n"
+                + "  \\__,_|_| |_|\\___|_|\\___| \n";
+
+        System.out.println("Hello I'm\n" + logo2);
+        System.out.println("What you want?");
+
+        Scanner in = new Scanner(System.in);
+        String filePath = "./data/duke.txt";
+        Storage storage = new Storage(filePath);
+        TaskList tasks;
+        boolean run = true;
```

Comment:
Consider renaming this variable according to the [Java Coding Standards](https://se-education.org/guides/conventions/java/basic.html) to make it sound more like a boolean. For example, `isRunning`